src/main/scala/firrtl/Emitter.scala

```diff
@@ -464,19 +467,6 @@ class VerilogEmitter extends SeqTransform with Emitter {
       "(2001). Use the SystemVerilog emitter instead.")
   }
   
-  /** 
-    * Store Emission option per Target
-    * Guarantee only one emission option per Target 
-    */
-  private[firrtl] class EmissionOptionMap[V <: EmissionOption](val df : V) extends collection.mutable.HashMap[ReferenceTarget, V] {
```

Comment:
Why did you delete this? I suspect that this is one of the causes for the `PresetSpec` failing. `Preset` relies on `EmissionOption`s.

Reply:
I see you moved it to compat, but it seems like you might have changed the implementation.